### PR TITLE
SC-1605

### PR DESCRIPTION
**Pull request by anmol2302**
**Ticket Ref:**  [SC-1605](https://project-sunbird.atlassian.net/browse/SC-1605)
1: Added ENV'S for cert registry service

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/sunbird_cert-registry-service.env [hunk 1]
```diff
@@ -1,3 +1,14 @@
 cert_service_base_url=http://cert-service:9000
 sunbird_es_host={{sunbird_es_host}}
 sunbird_es_port={{sunbird_es_port}}
+sunbird_cassandra_keyspace=sunbird
+sunbird_cassandra_username=cassandra
```
Comment (rjshrjndrn) [inline ansible/roles/stack-sunbird/templates/sunbird_cert-registry-service.env]: kindly variablize these values.

File: ansible/roles/stack-sunbird/templates/sunbird_cert-registry-service.env [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+sunbird_cassandra_password=password
+sunbird_cassandra_consistency_level=quorum
+{% if groups['cassandra-2'] is defined %}
+sunbird_cassandra_host={{groups['cassandra']|join(',')}}
+sunbird_cassandra_port=9042,9042,9042
```
Comment (rjshrjndrn) [inline ansible/roles/stack-sunbird/templates/sunbird_cert-registry-service.env]: can't we just have a single port ?

Comment (anmol2302) [reply ansible/roles/stack-sunbird/templates/sunbird_cert-registry-service.env]: no, it supports like this only @rjshrjndrn

File: ansible/roles/stack-sunbird/templates/sunbird_cert-registry-service.env (hunk 1 above)
Comment (anmol2302) [reply ansible/roles/stack-sunbird/templates/sunbird_cert-registry-service.env]: DONE

**Pull request merged**